lib/charms/lte_core_interface/v0/lte_core_interface.py

```diff
@@ -0,0 +1,245 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Library for the `lte-core` relation.
+
+This library contains the Requires and Provides classes for handling the `lte-core`
+interface.
+
+The motivation for creating this library comes from the need of relating charmed EPCs with
```

Comment:
Maybe something like the following is a bit easier to read:
```suggestion
The purpose of the library is to relate charmed EPCs with charmed enodeB simulators and user equipment (UEs), for testing.
```